### PR TITLE
add namespace support

### PR DESCRIPTION
**Pull request by HamadiSE**
skip networking tests if no orchestrator is found

### CODE REVIEW

**Review by edcdavid — COMMENTED** [review]
update the readme to indicate that test are executed per namespace as specified in the config file

**Review by edcdavid — COMMENTED** [review]
Probably not from this PR: Operator tests are passing with no operators detected. The operator is in the default namespace. PR in partner repo https://github.com/test-network-function/cnf-certification-test-partner/pull/37 needs to be checked in before this one

**Review by wying3 — APPROVED** [review]

**Review by edcdavid — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -22,8 +22,11 @@ In the diagram above:
 
 ## Test Configuration
 
-The Test Network Function support auto-configuration using labels and annotations, but also a static configuration using a file. The following sections describe how to configure the TNF via labels/annotation and the corresponding settings in the config file. A sample config file can be found [here](test-network-function/tnf_config.yml).
+The Test Network Function support auto-configuration using labels and annotations. The following sections describe how to configure the TNF via labels/annotation and the corresponding settings in the config file. A sample config file can be found [here](test-network-function/tnf_config.yml).
 
+### targetNameSpaces
+
+A single namespace should be specified in the [configuration file](test-network-function/tnf_config.yml). This namespace will be used by auto-configuration to discover the Pods under test. It's preferred not to run multiple tests in different namespaces in parallel, some tests are intrusive and can cause failures if two tests are running simultaneously.
```
Comment (jun-rh) [inline README.md:29]: to be consistent, autodiscovery instead of auto-configuration

Comment (HamadiSE) [reply README.md]: updated. please check new formulation

**Review by jun-rh — APPROVED** [review]

**Pull request merged**